src/locales/cy/default.yml

```diff
@@ -66,6 +66,7 @@ drivingLicence:
   retryMessageList2: rydych wedi rhoi eich rhif trwydded yrru yn gywir
   retryMessageList3: bod dyddiau a misoedd yn y drefn gywir yn eich dyddiad geni, dyddiad cyhoeddi a dyddiad dilys tan
   retryMessageList4: you’ve left the middle names section blank if you do not have any middle names
```

Comment:
Not you @Suad-Mohamed but @ElliotMurphyGDS Why is the above in english

Reply:
Because Its never used we can probably remove it